src/transport/Transport.java

```diff
@@ -0,0 +1,84 @@
+package transport;
+
+public abstract class Transport {
+    private final String brand;
+    private final String model;
+    private final int productionYear;
+    private final String productionCountry;
+    private String color;
+    private int maxSpeed;
+
+    public Transport(String brand, String model, int productionYear, String productionCountry, String color, int maxSpeed) {
+        if (brand == null || brand.isBlank()) {
+            this.brand = "default";
+        } else {
+            this.brand = brand;
+        }
```

Comment:
this.brand = validOrDefault(brand);

public static validOrDefault(String value){
  if (value == null || value.isBlank()) {
            return "default";
        } else {
            return value;
        }
}